draft-packages/collapsible/KaizenDraft/Collapsible/Collapsible.module.scss

```diff
@@ -64,16 +71,20 @@ $border-color: rgba($color-gray-600-rgb, 0.2);
   width: 100%;
   align-items: center;
   text-align: left;
-  background-color: white;
+  background-color: $color-white;
   border: none;
-  transition: background-color $ca-duration-rapid;
+  transition: background-color $animation-duration-rapid;
 
-  @include ca-padding(
-    $top: $ca-grid,
-    $end: $ca-grid,
-    $bottom: $ca-grid,
-    $start: $ca-grid
-  );
+  padding-top: $spacing-md;
+  padding-bottom: $spacing-md;
+  padding-right: $spacing-md;
+  padding-left: $spacing-md;
```

Comment:
If this is all the same padding could we just use `padding: $spacing-md`? Possibly the same thing on ln 113

Reply:
bahaha my brain didn't put that together - wow